components/bpel/org.wso2.carbon.bpel.ui/src/main/java/org/wso2/carbon/bpel/ui/bpel2svg/impl/SourceImpl.java

```diff
@@ -30,7 +30,7 @@
  * Source tag UI implementation
  */
 public class SourceImpl extends SourcesImpl implements SourceInterface {
-
+    public static final String SVG_NAMESPACE = "http://www.w3.org/2000/svg";
```

Comment:
Lets define the SVG_NAMESPACE in a single location and reuse it instead of defining it in each of the classes